src/Workspaces/Remote/Core/AbstractAssetProvider.cs

```diff
@@ -152,6 +149,18 @@ async Task<ImmutableArray<DocumentInfo>> CreateDocumentInfosAsync(DocumentChecks
         // TODO: do we need version?
```

Comment:
>// TODO: do we need version? [](http://example.com/codeflow?start=8&length=28)

TODO still in flux?

Reply:
want to talk to you in person about this.

Reply:
Might be a week or two before I'm back in the office due to that machine going down. I don't have any strong thoughts here, I just saw a todo and wondered if it was still valid.